xml/System.Resources.Extensions/DeserializingResourceReader.xml

```diff
@@ -64,8 +64,8 @@
         <Parameter Name="fileName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="fileName">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="fileName">The path and name of the resource file to be read. <c>filename</c> is not case-sensitive.</param>
```

Comment:
Not sure if it would read better without a self-reference

```suggestion
        <param name="fileName">The path and name of the resource file to be read. This parameter is not case-sensitive.</param>
```